### PR TITLE
Make usages of masculine pronouns gender-neutral

### PR DESCRIPTION
**Pull request by eakuefner**
<!-- Pointers to relevant developer documentation:

Contributing guide: https://github.com/coq/coq/blob/master/CONTRIBUTING.md

Test-suite: https://github.com/coq/coq/blob/master/test-suite/README.md

Changelog: https://github.com/coq/coq/blob/master/doc/changelog/README.md

Building the doc: https://github.com/coq/coq/blob/master/doc/README.md
Sphinx: https://github.com/coq/coq/blob/master/doc/sphinx/README.rst
doc_gram: https://github.com/coq/coq/blob/master/doc/tools/docgram/README.md

Overlays: https://github.com/coq/coq/blob/master/dev/ci/user-overlays/README.md

### CODE REVIEW

File: doc/sphinx/practical-tools/coq-commands.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Interactive use (coqtop)
 ------------------------
 
 In the interactive mode, also known as the Coq toplevel, the user can
```
Comment (jfehrle) [inline doc/sphinx/practical-tools/coq-commands.rst:22]:
```suggestion
In the interactive mode, also known as the Coq toplevel, the user can
```

**Comment by Alizter** [comment]
I've pushed my extra changes before I forget them anyway. Since it is the weekend, it might be some time before people review.

**Comment by eakuefner** [comment]
Thanks @Alizter!

File: doc/sphinx/practical-tools/coq-commands.rst (hunk 1 above)
Comment (eakuefner) [reply doc/sphinx/practical-tools/coq-commands.rst]: I've gone ahead and updated each instance that @Alizter changed to be plural and updated the title of the PR to better reflect the multiple changes.

**Review by jfehrle — APPROVED** [review]
LGTM

**Comment by Alizter** [comment]
@jfehrle Can you assign and merge?

**Comment by Alizter** [comment]
@SkySkimmer Since this is a trivial doc change, would you be willing to include it for 8.15?

**Comment by jfehrle** [comment]
@coqbot: merge now

**Pull request merged**